src/v2/Components/Address/AddressModalFields.tsx

```diff
@@ -42,7 +68,7 @@ export const AddressModalFields: React.FC = () => {
             />
           </Box>
         </Column>
-        <Column span={6}>
+        <Column span={12}>
```

Comment:
I _think_ by default `<Column>` spans 12, so no need for prop

Reply:
Column spans 1 by default. I'm not sure grid columns are even needed here anymore.